bindings/core/src/method/wallet.rs

```diff
@@ -318,6 +318,17 @@ pub enum WalletMethod {
         params: Vec<SendNftParams>,
         options: Option<TransactionOptions>,
     },
+    PrepareCreateDelegation {
+        params: CreateDelegationParams,
+        options: Option<TransactionOptions>,
+    },
+    #[serde(rename_all = "camelCase")]
+    PrepareDelayDelegationClaiming {
+        delegation_id: DelegationId,
+        reclaim_excess: bool,
+    },
+    #[serde(rename_all = "camelCase")]
+    PrepareDestroyDelegation { delegation_id: DelegationId },
```

Comment:
Please also add comments here like we have for all the others

Reply:
done